### DISCUSSION TITLE
How can I set defer for <script> tag? with async we have big performance problem!!!

### DISCUSSION THREAD

**Discussion by krasevych** [vercel/next.js#15167] [Help]
How can I set defer for <script> tag? with async we have big performance problem!!!

**Comment by IRediTOTO** [comment]
Lets look ay dynamic import with option ssr false

**Reply by krasevych** [reply]
I mean script like this one:
```<script async="" data-next-page="/_app" src="/_next/static/development/pages/_app.js?ts=1594752062649"></script>```
It should be like this:
```<script defer="" data-next-page="/_app" src="/_next/static/development/pages/_app.js?ts=1594752062649"></script>```

**Comment by c0b41** [comment]
they introduce `deferScripts` option #8980 but later they reverted because performance problem. 10143

edit: also similar discussion #11120

**Reply by krasevych** [reply]
oh, it's really sad
